import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/backupuploader/GcpBackupUploader.java

```diff
@@ -1,25 +1,129 @@
 package com.google.cloud.healthcare.imaging.dicomadapter.backupuploader;
 
+import com.google.auth.Credentials;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.ReadChannel;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.io.InputStream;
+import java.nio.channels.Channels;
+import java.util.List;
+import org.apache.http.client.utils.URIBuilder;
```

Comment:
I'd prefer to minimize the use of external libraries here, because the more we import the more likely that we get a flaky build. I think it should be easy to mimic the functionality of URIBuilder here?